package.json

```diff
@@ -118,5 +118,9 @@
   "bugs": {
     "url": "https://github.com/Lundalogik/lime-elements/issues"
   },
-  "homepage": "https://github.com/Lundalogik/lime-elements#readme"
+  "homepage": "https://github.com/Lundalogik/lime-elements#readme",
+  "dependencies": {
```

Comment:
We should not add dependencies as `dependencies`, the should be added as `devDependencies`